### PR TITLE
First Capstone project 

### PR DESCRIPTION
**Pull request by Mukumbuta**
### Features implementd
- The home page.
- The about page.

## Links
- The home page has a link in the menu to the about page.
- The logo in the header links to the home page.

## Mobile menu
- When the user clicks (or taps) the hamburger button on the header, the mobile menu appears over the page.
- The mobile menu has a close (X) button that closes the menu.

## Dynamic page
- The section "Featured speakers" is created dynamically in JavaScript.

### CODE REVIEW

**Review by MahmoudBakr23 — COMMENTED** [review]
# Hi @Mukumbuta 😃, 

As much as I appreciate your hard work on this project, I'm afraid that I can't perform a review on this pull request.

## **Invalid Submission**

- Please make sure to fix the linter errors.

_Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready._🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._

<hr>

_Invalid Code Review Request does not count into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -1,2 +1,51 @@
...
+
+- Major languages: HTML, CSS
+- Frameworks: N/A
+- Technologies used: Linters, Git, npm
+
+
+To get a local copy up and running follow these simple example steps.
+
+## Live demo
+- Link: https://www.loom.com/share/5a6ac145853c406785fd851e91e9fa93
```
Comment (EricMbouwe) [inline README.md]: - [ ] Your link doesn' t return a video. it may be the wrong one, let's fix that

File: README.md [hunk 2]
```diff
@@ -1,2 +1,51 @@
...
+
+
+## Deployment
+- Deployed using GitHub Pages
+- Link: https://mukumbuta.github.io/Capstone-Project-1/
```
Comment (EricMbouwe) [inline README.md:22]: - [ ] Seems like you opened the PR from the wrong branch or you are not done yet with your project
image

File: README.md [hunk 3]
```diff
@@ -1,2 +1,51 @@
...
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to @Kibirige-john 
+- GitHub: [@githubhandle](https://github.com/John-Kibirige)
```
Comment (EricMbouwe) [inline README.md]: - [ ] You should add acknowledgements to give appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015),

**Review by EricMbouwe — CHANGES_REQUESTED** [review]
Hi @Mukumbuta, 

Good job so far 🔥!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 🔥🎉
- Styles and mobile responsive ✔
- Linters ✔
- Use of semantic Html ✔
- Good implementation of JS feature ✔

# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Kandy-Peter — APPROVED** [review]
Hi @Mukumbuta 

### APPROVED 🟢 

Great implementation of all the requirements in this milestone. 👏
Your web app looks really fantastic. 🎉 
image
_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

##  Optional suggestion

 
# CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**